### ISSUE TITLE
Are we using a legacy/deprecated registry?

### ISSUE THREAD

**Issue by fperez** [jupyter/docker-stacks#120]
Howdy,

I'm far from a docker expert, so forgive me if this is noise, but I was trying to use our scipy-notebook image, and after running `docker run -d -p 8888:8888 jupyter/scipy-notebook`, the pull of the image worked (I didn't have it before), but I got this warning:

```
Status: Downloaded newer image for jupyter/scipy-notebook:latest
docker.io/jupyter/scipy-notebook: this image was pulled from a legacy registry.  Important: This registry version will not be supported in future versions of docker.
```

Not sure how serious it is, but I figured I'd flag it...

**Comment by parente** [comment]
The docker client knows to contact the primary docker hub server which redirects to one of its many registries. The current version of the docker registry API is v2, but I believe Docker Inc. still has some v1 registry servers in play.

That said, if you have an older version of the Docker client, pre 1.6 or so I believe, it can only talk to v1 registry servers. In that case, upgrade should help.

Bottom line: nothing that we're doing wrong here. The images are on Docker Hub which is "the place to be."

**Comment by fperez** [comment]
Great, thanks! Just wanted to be sure...
